examples-gallery/plot_sliding_block.py

```diff
@@ -0,0 +1,306 @@
+
+# %%
+""""
+sliding a block on a road
```

Comment:
Capitalize the title.

Reply:
Like This
Sliding a Block on a Road     or
Sliding a block on a road  ?
**Thanks!**

Reply:
Mimic the other examples: https://opty.readthedocs.io/latest/examples/